xdsl/dialects/experimental/fir.py

```diff
@@ -139,44 +138,6 @@ class ReferenceType(ParametrizedAttribute, TypeAttribute):
     name = "fir.ref"
     type: ParameterDef[Attribute]
 
-    def print_parameters(self, printer: Printer) -> None:
-        # We need this to pretty print a tuple and its members if
-        # this is referencing one, otherwise just let the type
-        # handle its own printing
-        printer.print("<")
-        if isinstance(self.type, TupleType):
-            printer.print("tuple<")
-            for idx, t in enumerate(self.type.types.data):
-                if idx > 0:
-                    printer.print(", ")
-                printer.print(t)
-            printer.print(">")
-        else:
-            printer.print(self.type)
-        printer.print(">")
```

Comment:
The default behaviour should now print the same thing as this custom implementation. This dialect is experimental so I am mostly hoping to be correct here as there are no tests.